proyecto prueba/Processes/NewCustomer.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using proyecto_prueba.Entities;
+
+namespace proyecto_prueba.Processes
+{
+    public class NewCustomer
+    {
+        public Customers NewCustomerMetodo()
+        {
+            Console.WriteLine("Registration of New Customer");
+            Console.WriteLine("CustomerId");
+            int sCustomerId = Convert.ToInt32(Console.ReadLine());
```

Comment:
Ok, we don't use Convert to cast datatypes, because its would launch an exception if there is not possible to convert the data.
So we use TryParse instead, like this:
if (!int.TryParse(Console.Readline(), out int customerId)
{
   return null://by example
}
//use the value converted.

Also we don't use the hungarian notation to name of variables, we just put a name understandable and hungarian notation could be for windows form controls.

Ps. I mean about the "s" to highlight as string